_posts/0000-01-02-OyaBellona.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide"
+---
+Your text
```

Comment:
The trouble with the rat race is that even if you win, you’re still a rat. - Lily Tomlin